www/install/php/Update-18.10.0.post.php

```diff
@@ -39,31 +39,54 @@
  * Generate random key for application key
  */
 $uniqueKey = md5(uniqid(rand(), true));
-$query = "INSERT INTO `informations` (`key`,`value`) VALUES ('appKey', '$uniqueKey')";
-$pearDB->query($query);
-$query = "INSERT INTO `informations` (`key`,`value`) VALUES ('isRemote', 'no')";
-$pearDB->query($query);
-$query = "INSERT INTO `informations` (`key`,`value`) VALUES ('isCentral', 'no')";
-$pearDB->query($query);
+$informationsData = [
+    [
+        'key' => 'appKey',
+        'value' => $uniqueKey
+    ],
+    [
+        'key' => 'isRemote',
+        'value' => 'no'
+    ],
+    [
+        'key' => 'isCentral',
+        'value' => 'no'
+    ]
+];
+foreach ($informationsData as $row) {
+    $query = "INSERT INTO `informations` (`key`,`value`) VALUES (:key, :value)";
+    $statement = $pearDB->prepare($query);
+    $statement->bindValue(':key', $row['key']);
+    $statement->bindValue(':value', $row['value']);
+    $statement->execute();
+}
```

Comment:
You should just sanitize the query for $query = "INSERT INTO `informations` (`key`,`value`) VALUES ('appKey', '$uniqueKey')";

As the other are hardcoded value there is no risks of injections. (and I think uniqueKey is also safe as its a result of  `md5(uniqid(rand(), true));`

Reply:
You're right, i restored it as it was before :)